### PR TITLE
Added permission for aws-sqs-plugin.

### PR DESCRIPTION
**Pull request by nickgrealy**
- [x] Hosting Request - https://issues.jenkins-ci.org/browse/HOSTING-189

### CODE REVIEW

File: permissions/plugin-aws-sqs.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "aws-sqs-plugin"
+paths:
+- "org/jenkins-ci/plugins/aws-sqs-plugin"
```
Comment (daniel-beck) [inline permissions/plugin-aws-sqs.yml]: Neither the `groupId` nor the `artifactId` part match what's specified in https://github.com/jenkinsci/aws-sqs-plugin/blob/master/pom.xml

Comment (nickgrealy) [inline permissions/plugin-aws-sqs.yml]: Sorry to be pedantic, but I'm not sure what you're asking. Are you saying:

**`artifactId` should match `name`?**

i.e. change
```Xml
<artifactId>aws-sqs</artifactId>
```
to
```Xml
<artifactId>aws-sqs-plugin</artifactId>
```

**`groupId` should match `paths`?**

i.e. change
```Xml
<project<groupId>io.relution.jenkins</groupId>
```
to
```Xml
<project><groupId>org.jenkins-ci.plugins</groupId>
```

Or vice versa (change the yml to match the pom)?

Comment (nickgrealy) [inline permissions/plugin-aws-sqs.yml]: Thanks for that, changes made.

- **pom.xml** - changed artifactId to [`aws-sqs-plugin`](https://github.com/jenkinsci/aws-sqs-plugin/blob/master/pom.xml#L15)
- **PR/plugin-aws-sqs.yml** - changed path to [match the instructions](https://github.com/jenkins-infra/repository-permissions-updater/pull/49/files#diff-f13797a5b8fe26791b26695e34639433R4)

Comment (daniel-beck) [inline permissions/plugin-aws-sqs.yml]: @nickgrealy We strongly prefer artifactIds that do not have the `-plugin` suffix. See https://wiki.jenkins-ci.org/display/JENKINS/Hosting+Plugins#HostingPlugins-Prerequisites

Comment (nickgrealy) [inline permissions/plugin-aws-sqs.yml]: Thanks for the suggestion. I removed `-plugin` from the `plugin-aws-sqs.yml` and `pom.xml` files.

**Pull request merged**